bundle/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -388,14 +388,20 @@ spec:
                 - --secure-listen-address=0.0.0.0:8443
                 - --upstream=http://127.0.0.1:8383/
                 - --logtostderr=true
-                - --v=10
```

Comment:
why this change?

Reply:
Again, I don't know, this is generated code.